examples/test_client2/test.py

```diff
@@ -41,8 +41,8 @@ def do_GET(self):
             self.send_response(200)
             self.send_header('Content-Type', 'text/plain')
             self.end_headers()
-            self.wfile.write('OAuth request token fetched and authorized;' +
-                ' you can close this window.')
+            self.wfile.write(bytes('OAuth request token fetched and authorized;' +
```

Comment:
I take it that you're running python3?  I'm surprised -- but pleased -- that this is the only change necessary to get things working on python3!

But I think you can just do `b'OAuth request token...'\nb' you can close...'` rather than calling `bytes`.  If so, that's more idiomatic python3, as I understand it.

Reply:
Oops, it looks like I tested the change with python3, but was intending to make changes that worked with python2. After rerunning it with python2, this part of the code works without any changes. I'll just remove these changes. 

If you did want this to be moved to python3 (which is unlikely based on what you've said), I'd be happy to do that.